# HG changeset patch
# User Vadim Yanitskiy <fixeria@osmocom.org>
# Date 1693502195 -25200
#      Fri Sep 01 00:16:35 2023 +0700
# Node ID 33866382d9994e07e7cb6eac65636f7f28e46b97
# Parent  b515a97e5dff252b24c41fecc139f8db19f6a637
ringtools/imy: fix duplicate definition of tdma_durations

diff -r b515a97e5dff -r 33866382d999 ringtools/imy/main.c
--- a/ringtools/imy/main.c	Mon Aug 28 03:52:53 2023 +0000
+++ b/ringtools/imy/main.c	Fri Sep 01 00:16:35 2023 +0700
@@ -9,7 +9,7 @@
 char *imy_filename;
 char melody_str_buf[MELODY_BUF_SIZE];
 unsigned beats_per_min = 120;
-unsigned tdma_durations[6][4];
+extern unsigned tdma_durations[6][4];
 FILE *outF;
 
 main(argc, argv)
